UI/Selenium/Pages/BookingDetailsPage.cs

```diff
@@ -9,5 +9,6 @@ public class BookingDetailsPage
 
         public static By CloseBookingFailureWindowButton = By.Id("btnTryAgain");
         public static By BookingConfirmedStatus = By.XPath("//div[@class='vh-created-booking'][text()='Confirmed']");
+        public static By SpecificBookingConfirmedStatus(string caseNumber) => By.XPath($"//div[@class='govuk-grid-column-full' and contains(.,'{caseNumber}') and contains(.,'Confirmed')]");
```

Comment:
I don't like this, I'd like to raise a request for the dev's to add an id if it can be worked out appropriately.  let's chat, but I will approve this code for now.

Reply:
The code is checking for specified case number "Confirmed" status.

Reply:
re my comments, accepted but worth getting a developer to add an id.